pkg/sqlite/studio.go

```diff
@@ -183,13 +291,16 @@ func (qb *StudioStore) Destroy(ctx context.Context, id int) error {
 	return qb.destroyExisting(ctx, []int{id})
 }
 
-// returns nil, nil if not found
+// returns nil, sql.ErrNoRows if not found
 func (qb *StudioStore) Find(ctx context.Context, id int) (*models.Studio, error) {
```

Comment:
The reason why I intentionally split the Find and find methods in my refactoring is to prevent sql.ErrNoRows errors from escaping out from the sqlite package. External code (such as resolvers) use the Find method, meaning that a graphql query for a missing studio returns null instead of an error. Internal code can use either method depending on whether the id is valid or not should be considered an error.

Reply:
Understood, sorry I didn't catch that reasoning from the code. I'll add it back in.